README.rst

```diff
@@ -115,6 +115,9 @@ The plugin API was upgraded from v0 to v1 in Tutor v13.2.0. This cookiecutter ge
         hooks.Filters.CONFIG_OVERRIDES.add_items(list(config.get("overrides", {}).items()))
 
     - Verify that the file contains no instance of "yourplugin" or "YOUR_PLUGIN". If it does, replace by your plugin name.
+    - In case the plugin has custom commands to be available from CLI, you will need to implement the CLI_COMMANDS filter according
+      the following reference: https://docs.tutor.overhang.io/reference/api/hooks/consts.html#tutor.hooks.Filters.CLI_COMMANDS
+      Once this is done, you will need to rename the command function or add a name=<plugin> argument to the @click.group call.
```

Comment:
Thanks for the PR @NeOneSoft ! A couple comments:
* It would be more elegant to link to the reference using an inline text link. For example, the **License** section below uses the text "AGPLv3" to link to the license instead of showing the link itself.
* I am note quite sure what `Once this is done, you will need to rename the command function or add a name=<plugin> argument to the @click.group call` is recommending. Perhaps adding an example would make it easier to understand?

Reply:
Thank you @kdmccormick for your review!!. I already update those changes and I'll be glad to attend your additional comments. Thanks again!!